### DISCUSSION TITLE
PAC ATLANTIC GEOLIA 13

### DISCUSSION THREAD

**Discussion by Cointel72** [fredlcore/BSB-LAN#555] [Q&A] [answered]
Good morning,
I want to drive an ATLANTIC GEOLIA 13 PAC
Is there a product compatible with this heat pump
Thank you for your reply

**Comment by fredlcore** [comment, marked as answer]
Taking a glance at the manual, it should be compatible, but to be 100% sure, you would have to open the cover and look for the controllers as depicted in our manual.

**Comment by Cointel72** [comment]
Thank you for your reply
I'll get back to you as soon as I have the info.
Cordially

**Comment by Cointel72** [comment]
> Le 11 mars 2023 à 18:02, ***@***.*** a écrit :
> 
> Thank you for your reply
> I'll get back to you as soon as I have the info.
> Cordially
> 
> 
>> Le 11 mars 2023 à 16:08, fredlcore ***@***.*** ***@***.***>> a écrit :
>> 
>> 
>> Taking a glance at the manual, it should be compatible, but to be 100% sure, you would have to open the cover and look for the controllers as depicted in our manual.
>> 
>> —
>> Reply to this email directly, view it on GitHub <https://github.com/fredlcore/BSB-LAN/discussions/555#discussioncomment-5276308>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6NR4WKUJ3ZPYEGNL5WUTJTW3SIQTANCNFSM6AAAAAAVXP223E>.
>> You are receiving this because you authored the thread.
>> 




Good morning,
I will come back to you to manage my heat pump.
I am attaching a photo of what I believe to be the Siemens controller.
Do you have a solution for this?
Sincerely
> 
￼

**Comment by fredlcore** [comment]
Yes, but please send me an e-mail to bsb(ät)code-it.de as mentioned on the main page. This is the discussion section of the project.
